CHANGELOG.md

```diff
@@ -1,10 +1,120 @@
+* `resource/vcd_nsxt_alb_edgegateway_service_engine_group` field `reserved_virtual_services` accepts
+  "0" as value ([#923](https://github.com/vmware/terraform-provider-vcd/pull/923))
+* Fixed a bug in `resource/vcd_vapp` that would prevent to Power off vApp when previous state was
+  `power_on=true` ([#932](https://github.com/vmware/terraform-provider-vcd/pull/932))
+
+### DEPRECATIONS
+* Deprecate `vcd_external_network` in favor of `vcd_external_network_v2` ([#903](https://github.com/vmware/terraform-provider-vcd/pull/903))
+* Deprecate `default_vm_sizing_policy_id` field in `vcd_org_vdc` resource and data source. This field is misleading as it
+  can contain not only VM Sizing Policies but also VM Placement Policies or vGPU Policies.
+  Its replacement is the `default_compute_policy_id` attribute ([#904](https://github.com/vmware/terraform-provider-vcd/pull/904))
+* Deprecate attribute `metadata` in favor of `metadata_entry` in the following data sources:
+  `vcd_catalog`, `vcd_catalog_media`, `vcd_catalog_vapp_template`, `vcd_independent_disk`, `vcd_network_direct`,
+  `vcd_network_isolated`, `vcd_network_isolated_v2`, `vcd_network_routed`, `vcd_network_routed_v2`, `vcd_org`,
+  `vcd_org_vdc`, `vcd_provider_vdc`, `vcd_storage_profile`, `vcd_vapp`, `vcd_vapp_vm`. ([#917](https://github.com/vmware/terraform-provider-vcd/pull/917))
+* Deprecate attribute `metadata` in favor of `metadata_entry` in the following resources:
+  `vcd_catalog`, `vcd_catalog_media`, `vcd_catalog_vapp_template`, `vcd_independent_disk`, `vcd_network_direct`,
+  `vcd_network_isolated`, `vcd_network_isolated_v2`, `vcd_network_routed`, `vcd_network_routed_v2`, `vcd_org`,
+  `vcd_org_vdc`, `vcd_vapp`, `vcd_vapp_vm`. ([#917](https://github.com/vmware/terraform-provider-vcd/pull/917))
+* Deprecate attribute `catalog_item_metadata` in favor of `metadata_entry` in the `vcd_catalog_item` resource
+  and data source. ([#917](https://github.com/vmware/terraform-provider-vcd/pull/917))
+* Deprecate `template_name` in favor of `vapp_template_id` in `vcd_vapp_vm` and `vcd_vm` to be able to use unique URNs instead
+  of catalog dependent names ([#931](https://github.com/vmware/terraform-provider-vcd/pull/931))
+* Deprecate `boot_image` in favor of `boot_image_id` in `vcd_vapp_vm` and `vcd_vm` to be able to use URNs instead
+  of catalog dependent names ([#931](https://github.com/vmware/terraform-provider-vcd/pull/931))
+* Deprecate `catalog_name` in favor of `vapp_template_id` or `boot_image_id`, which don't require a catalog name anymore ([#931](https://github.com/vmware/terraform-provider-vcd/pull/931))
+* Data source `vcd_nsxt_edge_cluster` deprecates `vdc` field in favor of 3 new fields to define
+  NSX-T Edge Cluster lookup scope - `vdc_id`, `vdc_group_id`, and `provider_vdc_id` ([#921](https://github.com/vmware/terraform-provider-vcd/pull/921))
```

Comment:
I can't seem to find a changelog entry for these three new fields? If they are new, they should be listed not only in deprecations section.

Reply:
This line was in "improvements" but I was told to move it to deprecations